playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -104,10 +104,11 @@ EDXAPP_HELP_MODAL_LINKS:
   - 'url': 'https://stanfordonline.zendesk.com/hc/en-us/articles/203009244-How-do-I-enroll-in-a-course-'
     'text': 'How to enroll in a course'
 EDXAPP_LANGUAGES:
-  - ['ar', 'Arabic']
-  - ['en', "English"]
-  - ['es-419', "Espa\u00f1ol"]
-  - ['vi', 'Vietnamese']
+  - ['ar', 'العربية']
+  - ['en', 'English']
+  - ['es-419', 'Español (Latinoamérica)']
+  - ['vi', 'Tiếng Việt']
+  - ['zh-cn', '中文 (简体)']
```

Comment:
is that: "Chinese (Mandarin)"?

Reply:
No it says Chinese (Simplified)

Reply:
oh ok.

btw, I love how "Simplified" uses more complicated characters 😁

Reply:
Just curious, what's the reason for changing the second item in the list?

Reply:
I just took the pairings from https://github.com/Stanford-Online/edx-platform/blob/master/lms/envs/common.py#L862
This also makes it consistent with edx.org account settings page.

Reply:
Cool.